readme.md

```diff
@@ -1,5 +1,5 @@
 <!-- DO NOT EDIT THIS FILE; it is auto-generated from readme.txt -->
-# AMP Plugin for WordPress
+# AMP
```

Comment:
It looks like you need to update your copy of `dev-lib` as this should not have been changed.

Reply:
Ah, thanks.

Reply:
It should be fixed with https://github.com/ampproject/amp-wp/pull/1734/commits/3b582a8516b5f45672c3ea6f4cb25c0aca8caa20